src/components/NavBar/NavBar.jsx

```diff
@@ -1,13 +1,39 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
you don't need to import react if you are using react >=v16

Reply:
Didn't know that. Thank you.